ui/src/data/source.tsx

```diff
@@ -391,43 +538,29 @@ function processEntitiesResponse(
     response.instances.slice(0, 100).forEach((instance) => {
       let ancestors: DataKey[] | undefined;
       let childCount: number | undefined;
-      const data: DataEntry = {
-        key: 0,
-      };
 
-      for (const k in instance) {
-        const v = instance[k];
-        // TODO(tjennison): Add support for computed columns.
-        if (k === "standard_concept") {
-          data[k] = v ? "Standard" : "Source";
-        } else if (!v) {
-          data[k] = "";
-        } else if (k === pathAttribute) {
-          if (v.stringVal) {
-            // TODO(tjennison): There's no way to tell if the IDs should
-            // be numbers or strings and getting it wrong will cause
-            // queries using them to fail because the types don't match.
-            // Figure out a way to handle this generically.
-            ancestors = v.stringVal.split(".").map((id) => +id);
-          } else if (v.stringVal === "") {
-            ancestors = [];
+      const data = processEntityInstance(
+        attributeID,
+        instance,
+        (key, value) => {
+          if (key === pathAttribute) {
+            if (value?.stringVal) {
+              // TODO(tjennison): There's no way to tell if the IDs should
+              // be numbers or strings and getting it wrong will cause
+              // queries using them to fail because the types don't match.
+              // Figure out a way to handle this generically.
```

Comment:
We should include the data type in the entity attributes definition -- I think that will address this. The indexing workflows need that anyway (at least longer-term, right now they're just assuming data types).